styles/sections/_nav.scss

```diff
@@ -154,27 +154,24 @@
   }
 }
 
-.site-nav ul {
+.site-nav-links {
   display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 64px 32px 0 32px;
+  text-align: left;
 
-  &.site-nav-links {
-    flex-direction: column;
-    gap: 24px;
-    padding: 64px 32px 0 32px;
-
-    @container body (min-width: 1024px) {
-      flex-direction: row;
-      gap: 16px;
-      padding: 0;
-    }
+  @container body (min-width: 1024px) {
+    flex-direction: row;
+    gap: 14px;
+    padding: 0;
+  }
 
-    @container body (min-width: 1240px) {
-      gap: 48px;
-    }
+  @container body (min-width: 1240px) {
+    gap: 48px;
   }
 
   &.social-nav {
```

Comment:
There aren't any elements with the `site-nav-links` class and the `social-nav` links class and it looks like you added new `.social-nav` styles else where, so these are no longer being used and can be removed I think

Reply:
We still both ul style classes, but I streamlined the styles a bit more and got rid of a couple styles we didn't really need like the row direction.

Reply:
Nevermind on that, I was on the main branch for a sec 😅 then realized my error. I think we do still need it (or would need to rework the section--I'll make a note in the #29 ticket to revisit then when we fix the background styles.